week2task/src/combankAccount/bank.java

```diff
@@ -0,0 +1,55 @@
+package combankAccount;
+
+public class bank {
+    private Account[] accounts;
```

Comment:
define this array as a dynamic array like that
` List<Account> accounts ;`
and inside the constructor of Bank , initialize this List with array list like that
`accounts = new ArrayList<>();`